changelog-template.hbs

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
The blank line here help define the stdout in the console which makes it easier to copy.